### PR TITLE
fix(canary-checker): upgrade to v0.38.183

### PR DESCRIPTION
**Pull request by paddatrapper**
Needed for the RBAC fixes

### CODE REVIEW

**Comment by paddatrapper** [comment]
Blocked by https://github.com/flanksource/canary-checker/issues/646

File: manifests/canary-checker.yaml [hunk 1]
```diff
@@ -145,6 +176,49 @@ metadata:
...
+      labels:
+        app.kubernetes.io/name: canary-checker-ui
+    spec:
+      containers:
+        - env:
+            - name: ORY_KRATOS_URL
+              value: null
+            - name: BACKEND_URL
+              value: ""
+          image: docker.io/flanksource/canary-checker-ui:v1.0.143
```
Comment (moshloop) [inline manifests/canary-checker.yaml:201]: Don't we need a way to configure this?

Comment (paddatrapper) [reply manifests/canary-checker.yaml:201]: Should it match the canary-checker version or is it independent?

Comment (paddatrapper) [reply manifests/canary-checker.yaml]: Done

**Review by Kaitou786 — APPROVED** [review]

**Pull request merged**